aws-mssql.yml

```diff
@@ -112,6 +112,15 @@ provision:
     type: boolean
     details: Whether deletion protection is enabled. The database cannot be deleted when this property is enabled.
     default: false
+  - field_name: option_group_name
+    type: string
+    default: null
+    nullable: true
```

Comment:
praise: it seems more reasonable to use the `nullable` type than the string. Thanks